### ISSUE TITLE
10$: Warn on return\n<div/>;

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#635]

**Comment by robminer6** [comment]
Do you mind if I claim this task? Also, what kind of details do you want in the warning message?

**Comment by strager** [comment]
> what kind of details do you want in the warning message?

The following code should report E0179:
```js
return
  <div>hi</div>;
```

**Comment by robminer6** [comment]
I claim this for-hire task. I expect payment after I complete this task. I will email the quick-lint-js team if I am assigned this task.
